pwa-devdocs/src/frequently-asked-questions/index.md

```diff
@@ -75,6 +75,22 @@ To let the storefront query a specific store view in Magento you need to add the
 
     -   Add `headers:{ Store: YOUR_STORE_CODE }`
 
+## Image component loads wrongly sized images from the srcSet
```

Comment:
```suggestion
## Image component loads images with the wrong size from the srcSet
```

Reply:
This adjustment wont pass the 60 char limit for headers I believe

Reply:
ah, you're right. we can leave as is then